documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -69,6 +69,24 @@ The Purpose definition (naming + description) and format within CAMARA follows t
 
 Purpose must be specified in the authorization request for a CAMARA Three-Legged Access Token, this is achieved by extending the `scope` parameter as defined in [Purpose as a Scope](CAMARA-Security-Interoperability.md#purpose-as-a-scope) section of the CAMARA Security and Interoperability Profile.
 
+## Controlling User-Interaction
+
+In CIBA User-Interaction in initiated by the API Producer if Consent is needed and NOT granted. The `login_hint` parameter is mandatory in CIBA and is used to identify the authentication devicde. Consent is captured out-of-band by e.g. sending a push notification, or by sending an SMS with a code or link, or by sending a message by some other means to the authentication devide. CAMARA does not mandate a communication method or any message format or content.
+
+In OIDC Authorization Code flow User-Interaction is initiated by the API Producer if consent is needed. The Authorization Server obtains End-User Consent/Authorization accorinding to [OIDC section 3.2.1.4](https://openid.net/specs/openid-connect-core-1_0.html#Consent).
```

Comment:
It would be advisable to maintain consistency with the rest of the document by using the same terms throughout. With regard to the term 'API producer', it is my understanding that when used in this context, the document states 'API provider' rather than 'API producer'.

Reply:
I searched the current document for "API pr" and found 4 hits, two "API provider", and two "API producer". I am going to change all hits to "API provider". Commonalities has "API Provider" in its glossary https://github.com/camaraproject/Commonalities/blob/main/documentation/Glossary.md
So, "API Provider" it is, right?! Should I change to C also in API consumer (4 occurrences)?

Reply:
I changed API producer to API Provider in the new section.
Should I change API producer everywhere? There is also API invoker, which should probably be API Consumer, right?

Reply:
Because of the Commonalities glossary I now changed API Invoker to API Consumer everywhere. Hope that is OK with everybody.
Also changed API [p|P]roducer to API Provider.
Also changed API [c|C]onsumer to API Consumer.

Reply:
I see you changed `API invoker` to `API Consumer`, but I'm afraid that if you look for just `Invoker`, there are 16 more coincidences in the document. I'd suggest replacing `Invoker` with `API Consumer`, and `invoker_callback` with `api_consumer_callback`.

Reply:
I changed Invoker to API consumer and invoker_callback to api_consumer_callback